app/code/Magento/Sitemap/etc/adminhtml/system.xml

```diff
@@ -51,7 +51,19 @@
                     <comment>Valid values range from 0.0 to 1.0.</comment>
                 </field>
             </group>
-            <group id="generate" translate="label" type="text" sortOrder="4" showInDefault="1">
+            <group id="store" translate="label" type="text" sortOrder="4" showInDefault="1" showInWebsite="1" showInStore="1">
+                <label>Store Url Options</label>
+                <field id="changefreq" translate="label" type="select" sortOrder="1" showInDefault="1" showInWebsite="1" showInStore="1" canRestore="1">
```

Comment:
Good practice is to increase `sortOrder` numbers by 10. This way extension developers are still able to extend the grid with their elements.

Reply:
@lbajsarowicz , thanks for reveiwing PR, I have updated the sort orders. Please help to verify.
Also can you please suggest how can I provide automated tests to PR.